README.md

```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
```

Comment:
Sorry I didn't notice this earlier-- I think you should change this title line to be more specific, e.g. "Lily's STEP Portfolio" or something, or just remove this line and the following blank line.